src/template.js

```diff
@@ -35,6 +38,30 @@ function logger(message, event) {
   log('[GTM-DEBUG] Search Insights > ' + message, event || '');
 }
 
+function clone(obj) {
```

Comment:
```suggestion
function shallowObjectClone(obj) {
```

Minor and non-blocking: I would have expected a `clone` function to do deep cloning by default, e.g. be capable of dealing with nested objects.

Reply:
Actually, what's the difference between this and `Object.assign({}, obj)`? (or `{...obj}`)
If the reason for that code that we cannot use a "recent" syntax in gtm code?